src/context/mod.rs

```diff
@@ -2,12 +2,9 @@ use std::sync::Arc;
 
 use crate::authoring::*;
 pub mod minimal;
-pub use minimal::Minimal;
```

Comment:
The update to version 1.75 appears to have made Clippy a bit smarter wrt. detecting unused imports, making the changes in mod.rs necessary. I first updated from 1.74 this morning, so I hadn't noticed before